mParticle-RevealMobile/MPKitRevealMobile.m

```diff
@@ -16,27 +16,51 @@
 //  limitations under the License.
 //
 
-#import "MPKitCompanyName.h"
+#import "MPKitRevealMobile.h"
 #import "mParticle.h"
 
+// This rather convoluded set of #defines allows you to pass in environment variables
+// for buddy build but not require them if you're not.
+//
+// Do to the XCode mechanism provided, you will get directive defined as empty instead
+// of undefined if there is no environment variable defined. This is seems overly complex
+// for such a simple task, but it handles all of the cases as far as I can see.
+#define STRINGIFY(str) @#str
+#define EXPAND_STRING(str) STRINGIFY(str)
+#define NO_OTHER_MACRO_STARTS_WITH_THIS_NAME_
+#define IS_EMPTY(name) defined(NO_OTHER_MACRO_STARTS_WITH_THIS_NAME_ ## name)
+#define EMPTY(name) IS_EMPTY(name)
+
+#define DO_EXPAND(VAL)  VAL ## 1
+#define EXPAND(VAL)     DO_EXPAND(VAL)
+
+#if !defined(USE_DEBUG_BEACONS) || (EXPAND(USE_DEBUG_BEACONS) == 1)
```

Comment:
I'm not sure these pre-compiler directive will work in this context. The definitions would have to be added to the `Pods` project, not the host app project in order for the the compiler to take these into consideration.